ocs_ci/templates/openshift-infra/logging-deployment/clusterlogging_operator/instance.yaml

```diff
@@ -15,17 +15,17 @@ spec:
           cpu: "600m"
           memory: "4Gi"
       storage:
-        storageClassName: "gp2"
+        storageClassName: "ocs-storagecluster-ceph-rbd"
```

Comment:
this is fine for now since my pr is not yet merged, but we can use get_defaultstorageclass() function in future.

Reply:
Since gp2 is the only default storage class, for now i am going with the same code.
```$ oc get sc
NAME                          PROVISIONER                             AGE
gp2 (default)                 kubernetes.io/aws-ebs                   2d9h
ocs-storagecluster-ceph-rbd   openshift-storage.rbd.csi.ceph.com      2d6h
ocs-storagecluster-cephfs     openshift-storage.cephfs.csi.ceph.com   2d6h
openshift-storage.noobaa.io   openshift-storage.noobaa.io/obc         2d6h
```